src/NetAnalyzers/UnitTests/Microsoft.NetCore.Analyzers/Runtime/ProvideCorrectArgumentsToFormattingMethodsTests.cs

```diff
@@ -446,7 +960,18 @@ class Test
 
     void M1(string param)
     {
-        var a = MyFormat("""", 1);
+        // Too many args
+        var s1 = MyFormat(""{0}"", 1, 2);
+        // Too many args and missing format index
+        var s2 = MyFormat(""{1}"", 1, 2, 3);
+
+        // Not enough args
+        var s3 = MyFormat(""{0} {1}"", 1);
+        // Not enough args and missing format index
+        var s4 = MyFormat(""{0} {2}"", 1);
```

Comment:
Same comment regarding the separate descriptors.

Reply:
yep, lets add a new descriptor for this case.

---
In reply to: [508700027](https://github.com/dotnet/roslyn-analyzers/pull/4226#discussion_r508700027) [](ancestors = 508700027)